tasks/cluster-enable-disable.yml

```diff
@@ -0,0 +1,26 @@
+- name: Enable or disable configured cluster services on boot
+  service:
+    name: "{{ item }}"
+    enabled: "{{ ha_cluster_start_on_boot }}"
+  loop:
+    - corosync
+    - corosync-qdevice
+    - pacemaker
```

Comment:
These should be defined as a list `__ha_cluster_services` in `vars/main.yml`

Reply:
hm - on the other hand, I see several places below where you want to treat each service individually - I think it may still be worth it to define all of the services in a single list in cases where you need to operate on all of them in a single task

Reply:
fixed in 1c2a9bf